featuretests/cmd_juju_controller_test.go

```diff
@@ -133,7 +144,7 @@ models:
   life: alive
   status:
     current: available
-    since: .*
+    since: just now
```

Comment:
Hey @timClicks. This line has been causing the race test suite to fail regularly.
See example [here](https://jenkins.juju.canonical.com/view/Unit%20tests/job/RunUnittests-race-amd64/1597/testReport/junit/github/com_juju_juju_featuretests/TestPackage/)